### DISCUSSION TITLE
I open-sourced app made with re-frame

### DISCUSSION THREAD

**Discussion by madvas** [day8/re-frame#699] [Show and tell]
Hi,

Sorry, this isn't issue, but I didn't know how to contact.

I just launched and open-sourced middle-size web app built on top of re-frame. First of all, huge thanks for this exceptional framework, I really enjoyed it, it's really something!

I write, because maybe you'd want to list it somewhere in "Examples" or "Showcase" section, so other people can use it as an example or reuse some snippets.

Source code at: https://github.com/madvas/fractalify/
Web app at: http://fractalify.com/

Thanks again!
Matus

**Comment by danielcompton** [comment]
Thanks! I've added this section to the README. fractalify looks pretty neat too!

**Comment by mike-thompson-day8** [comment]
Sorry I'm so late to this,  but I also added to the Wiki:  https://github.com/Day8/re-frame/wiki/External-Resources
